src/commissaire/models/__init__.py

```diff
@@ -241,6 +241,11 @@ def _coerce(self):
 class Network(Model):
     """
     Representation of a network.
+
+    .. note::
+
+       This model is similar to ContainerManagerConfig. The options
+       attribute holds configuration items used to define network options.
```

Comment:
I suggest dropping the network options sentence here.  I found it a little confusing since `Network` has no `options` attribute.